.github/actions/delete-deployments/action.yml

```diff
@@ -31,7 +31,7 @@ runs:
   steps:
     - uses: pnpm/action-setup@a3252b78c470c02df07e9d59298aecedc3ccdd6d # v3.0.0
       with:
-        version: ^8.0.0
+        version: ^9.0.0
```

Comment:
you can remove this field entirely, then it'll infer the version from package.json

Reply:
Doesn't appear to work

Reply:
It works based off the `packageManager` field in the `package.json`, which we do not have. Should be fine to leave the version pinned in the above file or update the `package.json`.

From pnpm/action-setup [readme](https://github.com/pnpm/action-setup/blob/master/README.md?plain=1#L11-L13)
> Version
> **Optional** when there is a [`packageManager` field in the `package.json`](https://nodejs.org/api/corepack.html). Otherwise, this field is **required** It supports npm versioning scheme, it could be an exact version (such as `6.24.1`), or a version range (such as `6`, `6.x.x`, `6.24.x`, `^6.24.1`, `*`, etc.), or `latest`.

Reply:
Yeah that's my bad, lets keep the version specifier in `action-setup`